src/app/(with-navbar)/(with-doc-nav)/docs/components/[type]/page.tsx

```diff
@@ -12,16 +14,34 @@ import { getHighlighter, highlight } from "@lib/shiki";
 import {
   openGraphImageSize,
   sharedOpenGraphMetadata,
+  flattenedDocsItems,
 } from "src/docs/constants";
-import { CONTENT_DIR, DocArticle } from "@components/InstallationDocs";
-import { componentTypes, mdxDocs } from "@examples";
+import { CONTENT_DIR, DocArticle } from "@components/MdxComponents";
+import { mdxDocs } from "src/docs/content/components";
+import { componentTypes } from "@examples";
 
 type ComponentPageProps = {
   params: {
     type: string;
   };
 };
 
+function findNeighbours(slug: string[]) {
```

Comment:
I see that we defining this function in here and also in the `/docs/[...slug]/page`. Wouldn't be better to put this function to some kind of utils folder and define it only once?

Reply:
Actually wanted to check if possible to reuse by merging these two routes, keeping in notes

Reply:
Update, two routes merge almost finished in followup branch, PR soon (did not want to clutter this one)